services/sql-migrator/migrator.go

```diff
@@ -0,0 +1,134 @@
+package migrator
+
+import (
+	"bytes"
+	"database/sql"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+
+	"text/template"
+
+	"github.com/golang-migrate/migrate/v4"
+	"github.com/golang-migrate/migrate/v4/database"
+	"github.com/golang-migrate/migrate/v4/database/postgres"
+	bindata "github.com/golang-migrate/migrate/v4/source/go_bindata"
+	"github.com/golang-migrate/migrate/v4/source/httpfs"
+)
+
+// Migrator is responsible for migrating postgres tables
+type Migrator struct {
+	// MigrationsTable is name of the table that holds current migration version.
+	// Each migration set requires a separate MigrationsTable.
+	MigrationsTable string
+
+	// Handle is the sql.DB handle used to execute migration statements
+	Handle *sql.DB
+}
+
+// Migrate migrates database schema using migration SQL scripts.
+func (m *Migrator) Migrate(migrationsDir string) error {
+	destinationDriver, err := m.getDestinationDriver()
+
+	path := filepath.Join("/", migrationsDir)
```

Comment:
This line of code is to add `/` at the beginning if missing, is my understanding correct?

Reply:
Yes